packages/backend-core/src/redis/redis.ts

```diff
@@ -291,23 +291,16 @@ class RedisWrapper {
       return acc
     }, {} as Record<string, any>)
 
-    const luaScript = `
-        for i, key in ipairs(KEYS) do
-            redis.call('MSET', key, ARGV[i])
-            ${
-              expirySeconds !== null
-                ? `redis.call('EXPIRE', key, ARGV[#ARGV])`
-                : ""
-            }
-        end
-        `
-    const keys = Object.keys(dataToStore)
-    const values = Object.values(dataToStore)
+    const pipeline = client.pipeline()
```

Comment:
Pipelines aren't guaranteed to be atomic. I don't believe that is important for this code, the worse that can happen is the expiry is overwritten with a slightly different time, but just wanted to point it out in case you can think of a reason it would matter.

Reply:
Pipelines are meant to be leaner, I think they should be fine for this use case